### DISCUSSION TITLE
Is there a way to send multiple files in reply?

### DISCUSSION THREAD

**Discussion by jsstrn** [telegraf/telegraf#1487] [Q&A] [answered]
If I want to have the bot reply with only one file I can use [`replyWithDocument`](https://telegraf.js.org/classes/context.html#replywithdocument).

Is there a way for me to reply with multiple files?

**Comment by Latvels** [comment, marked as answer]
Hi! Try using this method https://telegrafjs.org/#/?id=sendmediagroup

**Reply by jsstrn** [reply]
Thanks for the answer. It appears I can also use `replyWithMediaGroup` within `context` to avoid having to pass in `chatId`.
